docs/web-component.md

```diff
@@ -0,0 +1,881 @@
+      this.LuigiClient.linkManager().openAsDrawer(url, {header:true, backdrop:true, size:'s'});
+    });
+
+    EcEvent.removeAllListeners();
+    EcEvent.register("alertBox",  this.showAlert.bind(this));
+    EcEvent.register("dialogUrl",  event => this.dialogUrl = event.detail);
+    EcEvent.register("splitUrl",  event => this.splitUrl = event.detail);
+    EcEvent.register("drawerUrl",  event => this.drawerUrl = event.detail);
+    EcEvent.register("navigationUrl",  event => this.navigationUrl = event.detail);
+  }
+
+  addConfirmationModal(){
+    const button = this.shadowRoot.querySelector('#bConModal');
+    button.addEventListener('click', (event) => {
+      this.LuigiClient.uxManager().showConfirmationModal({
+        header: "Confirmation",
+        body: "Are you sure you want to do this?",
+        buttonConfirm: "Yes",
+        buttonDismiss: "No"
+      });
+      event.stopPropagation();
+    });
+  }
+
+  addClickNavButton(selector, valueToCheck, clickFn){
+    const button = this.shadowRoot.querySelector(selector);
+    button.addEventListener('click', (event) => {
+      if (!this[valueToCheck] || this[valueToCheck].length === 0){
+        this.LuigiClient.uxManager().showAlert({
+          text: 'Please insert an ulr in the input element',
+          type: 'error'
+        });
+        return;
+      }
+
+      clickFn(this[valueToCheck]);
+      event.stopPropagation();
+    }, false);
+  }
+
+  addClickButtonBusy(){
+    this.bHideBusy = this.shadowRoot.querySelector('#bHideBusy');
+    this.bShowBusy = this.shadowRoot.querySelector('#bShowBusy');
+    this.bShowBusy.style.display = 'none';
+    this.bHideBusy.addEventListener('click', (event) => {
+      this.toggleBusy('busyIndicatorExample', false);
+      event.stopPropagation();
+    }, false);
+    this.bShowBusy.addEventListener('click', (event) => {
+      this.toggleBusy('busyIndicatorExample', true);
+      event.stopPropagation();
+    }, false);
+  }
+
+  showAlert(event){
+    let alertType = event.detail;
+    if (this.LuigiClient) {
+      this.LuigiClient.uxManager().showAlert({
+        text: 'Hello from uxManager in Web Component',
+        type: alertType
+      });
+    }
+    event.stopPropagation();
+  }
+
+  toggleBusy(id, show){
+    EcEvent.fire(id, show);
+    const showButtonStyle = show ? 'none' : 'block';
+    const hideButtonStyle = show ? 'block' : 'none';
+
+    this.bShowBusy.style.display = showButtonStyle;
+    this.bHideBusy.style.display = hideButtonStyle;
+    this.showBusy = show;
+  }
+}
+
+import './f_selectbox.js';
+```
+
+In this file we have the following elements:
+- HTML template of the components where we have basically 4 sections
+- An injected <fundamental-list-box web component
+- An injected <f_busy web component
+- An injected <fundamental-input used 4 times
+- We created an example of event handler (file wcEvent.js) that is going to be used for the web components communication.
+
+###File f_selectbox.js
```

Comment:
```suggestion
### File f_selectbox.js
```

Reply:
I've already changed in the newest version